patch/patcher.go

```diff
@@ -531,12 +532,29 @@ func (p *Patcher) patchIdent(id *ast.Ident, obj types.Object) {
 			if v.Tag == nil {
 				v.Tag = &ast.BasicLit{}
 			}
-			v.Tag.Value = "`" + strings.TrimSpace(strings.Trim(v.Tag.Value, "` ")+" "+tags) + "`"
+
+			v.Tag.Value = "`" + mergeTags(v.Tag.Value, tags) + "`"
 			log.Printf("Add tags:\t%q.%s %s", obj.Pkg().Path(), id.Name, v.Tag.Value)
 		}
 	}
 }
 
+// Override tags
+func mergeTags(oldTag, newTag string) string {
+	oldTags := strings.Split(strings.TrimSpace(strings.Trim(oldTag, "` ")), " ")
```

Comment:
Did you mean to have the trailing space on the trim?

```suggestion
	oldTags := strings.Split(strings.TrimSpace(strings.Trim(oldTag, "`")), " ")
```